pandas/io/json/json.py

```diff
@@ -129,6 +129,10 @@ def _write(self, obj, orient, double_precision, ensure_ascii,
                date_unit, iso_dates, default_handler):
         if not self.index and orient == 'split':
             obj = {"name": obj.name, "data": obj.values}
+        if orient == 'records':
```

Comment:
I'll backtrack on my previous comment - probably OK to do this here since we already have similar code in the `FrameWriter`. 

With that said, I don't think accessing `.values` is appropriate here. Is there a way to leverage a call to `to_dict` in a fashion similar to what's done in the `FrameWriter` and use that result there?

Reply:
oh, okay, actually i did this `.values` because when having `orient=='records'`, i think series should behave similar to `FrameWriter`, but what i did now for me is identical to `.values`, but just avoid the warning...  i will take a look at `to_dict` to see if i can use it here.

Reply:
@WillAyd Hi, i check the FrameWriter, and for the case when `orient="records"`, obj should return a list of dictionaries in which dict key should be the same, and dict values are obj.values... i haven't solved how to appropriately leverage a call to `to_dict` here... can you give me some tips? thanks very much!